.gitignore

```diff
@@ -6,3 +6,7 @@ skema_venv/*
 __pycache__/
 instance?
 .pytest_cache/
+
+
+#Exclude venv folder
+venv/*
```

Comment:
It looks like you named you virtualenv "env" not "venv". Can you fix this file, and then delete all of the env/* files from this PR please?